src/commands/setup.rs

```diff
@@ -117,10 +117,25 @@ impl Setup {
                         };
                         continue;
                     }
+
+                    // parse isolation option here so we can avoid unwrapping and storing it unecessarily
+                    let isolation_config =
+                        match network.options.as_ref().and_then(|map| map.get("isolate")) {
+                            Some(isolation) => match isolation.parse() {
+                                Ok(isolation) => isolation,
+                                Err(err) => {
+                                    bail!("{} could not parse isolation option", err);
```

Comment:
At higher level setup calls we are using wrapping via ` return Err(std::io::Error::new(` throughout the code, could we wrap this in error instead of `bail` for consistency.

Reply:
ok, I will do the same for the other ignored errors below